channels/ON-CHAIN.md

```diff
@@ -222,24 +250,29 @@ the initiator and responder (before the close) and the fee.
 
 This transaction MUST have valid signatures of all involved parties.
 
+This transaction MUST NOT be disputed and any ongoing dispute MUST be considered
```

Comment:
Shouldn't it be "CAN NOT"?

Reply:
MUST NOT means »is not allowed to«

CAN NOT is also not defined in https://tools.ietf.org/html/rfc2119